src/config/env.js

```diff
@@ -28,4 +29,8 @@ module.exports = Object.freeze({
     // Mail
     mail_from: process.env.MAIL_FROM,
     mail_from_password: process.env.MAIL_FROM_PASSWORD,
+
+    // File upload
+    cloudinary_url: process.env.CLOUDINARY_URL,
+    upload_folder: path.join(path.join(__dirname, "../.."), process.env.UPLOAD_FOLDER || "public")
```

Comment:
```suggestion
    upload_folder: path.join(path.join(__dirname, "../.."), process.env.UPLOAD_FOLDER || "static")
```

Making the default folder name be "static" probably clears some confusion. Also, does it need to exist beforehand, or will it create if not exist?

Reply:
it will create if the folder does not exist